### ISSUE TITLE
Backup & Export, Import

### ISSUE THREAD

**Issue by ghost** [tiancihe/bookmark-manager#14]
Is it possible to bring this feature through your extension, as in chromium-based browsers?

**Comment by tiancihe** [comment]
Yes, i refered to [these docs](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions) while i was developing this extension

> The technology for extensions in Firefox is, to a large extent, compatible with the extension API supported by Chromium-based browsers (such as Google Chrome, Microsoft Edge, Opera, Vivaldi). In most cases, extensions written for Chromium-based browsers run in Firefox with just a few changes.

**Comment by tiancihe** [comment]
Did some research in regard to this feature, found out that there are no standard api for this. To implement this feature requires some additional xml parsing and bookmark tree serializing techniques. In addition, information such as date added and last time modified cannot be restored due to current api limitations.

May i ask why do you want this feature, i rarely use this import/export feature in the default bookmark manager except for the first time migration from other browsers. You can describe your use case, there might be other alternative solutions to this.
